### PR TITLE
ja2-stracciatella: init 0.15.1

### PR DESCRIPTION
**Pull request by bendlas**
This is the open-source version of the venerable Jagged Alliance 2.

Stracciatella means almost vanilla. This version is only changed for
fixes, stability improvements and to host mods.

The original data files are non-free and have to be provided by the user.

###### Motivation for this change
More linux games 8-)

###### Things done
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS (should run there too)
   - [x] Linux
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by bendlas** [comment]
Removed debug info due to https://github.com/NixOS/nixpkgs/issues/22840

File: pkgs/games/ja2-stracciatella/default.nix [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+stdenv.mkDerivation rec {
+  version = "0.15.1";
+  name = "ja2-stracciatella-${version}";
+  src = fetchFromGitHub {
+    owner = "ja2-stracciatella";
+    repo = "ja2-stracciatella";
+    rev = "v${version}";
+    sha256 = "0r7j6k7412b3qfb1rnh80s55zhnriw0v03zn5bp3spcqjxh4xhv1";
+  };
+  enableParallelBuild = true;
```
Comment (fpletz) [inline pkgs/games/ja2-stracciatella/default.nix]: This should be `enableParallelBuilding`.

File: pkgs/games/ja2-stracciatella/default.nix [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  outputs = [ "out" ];
```
Comment (fpletz) [inline pkgs/games/ja2-stracciatella/default.nix]: Not needed. Please remove.

**Review by fpletz — CHANGES_REQUESTED** [review]

**Comment by bendlas** [comment]
fixed & force pushed

**Review by fpletz — APPROVED** [review]

**Pull request merged**